src/providers/http.js

```diff
@@ -12,7 +12,7 @@ module.exports = () => {
      * @return {Promise} A promise that resolves to the contents of the file
      */
     getFile (host, path) {
-      return axios(`${host}/${path}`, {
+      return axios(`http://${host}/${path}`, {

```

Comment:
The `host` could already contain `http://` or `https://`. We should add a regex that checks whether it is missing and if so, add `http://`

Reply:
Yeah apparently it is failing. I think it's a good idea with a regex solution. I'll try to fix it